website/content/docs/operators/deployment/schedulers/kubernetes-helm.md

```diff
@@ -0,0 +1,231 @@
+$ ./install-helm.sh
+```
+
+## Installing Helm in your Kubernetes cluster
+
+To run Helm on Kubernetes, you need to first make sure that [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) is using the right configuration context for your cluster. To check which context is being used:
+
+```bash
+$ kubectl config current-context
+```
+
+If the context is correct, then you can get Helm running using just one command:
+
+```bash
+$ helm init
+```
+
+If the output of that command includes `Happy Helming!` then Helm is ready to go.
+
+## Installing Heron on Kubernetes
+
+Once you've installed the Helm client on your machine and gotten Helm running in your Kubernetes cluster, you need to make your client aware of the `heron-charts` Helm repository, which houses the chart for Heron:
+
+```bash
+$ helm repo add heron-charts https://storage.googleapis.com/heron-charts
+"heron-charts" has been added to your repositories
+```
+
+Now you can install the Heron package:
+
+```bash
+$ helm install heron-charts/heron
+```
+
+This will install Heron and provide the installation with a random name like `jazzy-anaconda`. To provide the installation with a name, such as `heron-kubernetes`:
+
+```bash
+$ helm install heron-charts/heron \
+  --name heron-kubernetes
+```
+
+### Specifying a platform
+
+The default platform for running Heron on Kubernetes is [Minikube](#minikube). To specify a different platform, you can use the `--set platform=PLATFORM` flag. Here's an example:
+
+```bash
+$ helm install heron-charts/heron \
+  --set platform=gke
+```
+
+The available platforms are:
+
+Platform | Tag
+:--------|:---
+[Minikube](#minikube) | `minikube`
+[Google Kubernetes Engine](#google-kubernetes-engine) | `gke`
+[Amazon Web Services](#amazone-web-services) | `aws`
+[Bare metal](#bare-metal) | `baremetal`
+
+#### Minikube
+
+To run Heron on Minikube, you need to first [install Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/). Once Minikube is installed, you can start it by running `minikube start`. Please note, however, that Heron currently requires the following resources:
+
+* 7 GB of memory
+* 5 CPUs
+* 20 GB of disk space
+
+To start up Minikube with the minimum necessary resources:
+
+```bash
+$ minikube start \
+  --memory=7168 \
+  --cpus=5 \
+  --disk-size=20g
+```
+
+Once Minikube is running, you can then install Heron in one of two ways:
+
+```bash
+# Use the Minikube default
+$ helm install heron-charts/heron
+
+# Explicitly select Minikube
+$ helm install heron-charts/heron \
+  --set platform=minikube
+```
+
+#### Google Kubernetes Engine
+
+The resources required to run Heron on [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) vary based on your use case. To run a basic Heron cluster intended for development and experimentation, you'll need at least:
+
+* 3 nodes
+* [n1-standard-2](https://cloud.google.com/compute/docs/machine-types#standard_machine_types) machines
+* 2 SSDs per machine
```

Comment:
n1-standard-4 and there is no need for 2 SSDs per machine. Since we use stateful sets, there is no need for it.